lib/ladle/steward_notifier.rb

```diff
@@ -8,11 +8,12 @@ def initialize(repository_name, pull_request)
     def notify(stewards_changes)
       notified_users = []
       stewards_changes.each do |github_username, changes_view|
-        user = User.find_by_github_username(github_username)
-
-        if user && ! user_has_been_notified?(user)
-          send_email(user, changes_view)
-          notified_users << user
+        if github_username.is_a?(Array)
```

Comment:
You can use `is_an?` or `kind_of?` to follow English grammar better.

Reply:
@magneland @limbov8 Hi! 👋    Sorry about the drive-by comment, but these two very similar calls to `notify_user` caught my eye. You could actually remove the type-check and conditional here by using `Array()`:

```ruby
Array(github_username).each do |username|
  notify_user(notified_users, username, changes_view)
end
```